tests/Happyr/LinkedIn/LinkedInTest.php

```diff
@@ -35,6 +36,19 @@ public function testConstructor()
             'Expect the API secret to be set.');
     }
 
+    public function testInitProperlyAssignsPassedInStorage()
```

Comment:
Good call. I think using an `IlluminateSessionStorage` object is a good measure. But I think leaving the assertion as `$this->assertEquals($sessionStorage, $ln->getStorage());` is more prudent as it will make sure that the actual variable we passed to the constructor got assigned to the member variable. `$this->assertInstanceOf(...)` only guarantees that the member variable is of the same instance as the passed in variable, not that the variable was retained properly by the object. What do you think?